src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,30 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.dao.ManufacturerDaoImpl;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+
     public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = new ManufacturerDaoImpl();
+        Manufacturer manufacturer1 = new Manufacturer(null, "Lincoln", "USA");
+        Manufacturer manufacturer2 = new Manufacturer(null, "Ford", "USA");
+        Manufacturer manufacturer3 = new Manufacturer(null, "Audi", "Germany");
+        Manufacturer manufacturer4 = new Manufacturer(null, "BMW", "Germany");
```

Comment:
It would be easier to understand if you call them by their manufacturers' names.
```suggestion
        Manufacturer lincoln = new Manufacturer(null, "Lincoln", "USA");
        Manufacturer ford = new Manufacturer(null, "Ford", "USA");
        Manufacturer audi = new Manufacturer(null, "Audi", "Germany");
        Manufacturer bmw = new Manufacturer(null, "BMW", "Germany");
```